test.js

```diff
@@ -195,7 +194,9 @@ const names = {
 };
 
 const testFile = (t, type, name) => {
-	t.is(check(type, name), type);
+	const {ext, mime} = check(type, name);
+	t.is(ext, type);
+	t.truthy(mime);
```

Comment:
To be entirely correct, this should test that it's a string.
```suggestion
	t.is(typeof mime, 'string');
```